packages/vscode-extension/messages.json

```diff
@@ -18,8 +18,8 @@
 	"COMMAND_TRIGGERED": "'{0}' has been triggered.",
 	"COMMAND_WARNING": "'{0}' has finished.",
 
-	"COMPARE_FILE_COMPARE_FILE_WITH_ACCOUNT_FILE": "Compare with Account File",
 	"COMPARE_FILE_COMPARING_FILE": "Comparing file...",
+	"COMPARE_FILE_EDITOR_LABEL": "Account Copy ⟷ Local Copy",
```

Comment:
I see it here. Yes, i think we can change it to an arrow only on the Local Copy side.

Reply:
It looks good to me as it is

Reply:
Maybe we could have the file name into the tab title instead of this string? Is there any reason for not doing it?

Reply:
It was suggested to go with this. What do you thin Karl?
filename.js (Account Copy  ⟷ Local Copy)

Reply:
The final decision is to go with this format...
`filename.js (Account <-> Local)`